lib/bot.js

```diff
@@ -111,6 +111,7 @@ class Bot {
   parseText(message) {
     const text = message.mentions.reduce((content, mention) => (
       content.replace(`<@${mention.id}>`, `@${mention.username}`)
+             .replace(`<@!${mention.id}>`, `@${mention.username}`)
```

Comment:
Users *without* nicknames are pinged as `<@id>`, users with are pinged as `<@!id>`. Not sure why there needs to be a difference, but there is one. Both lines are needed.